Userland/Utilities/shot.cpp

```diff
@@ -1,32 +1,93 @@
 /*
  * Copyright (c) 2021, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2021, Aziz Berkay Yesilyurt <abyesilyurt@gmail.com>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #include <AK/Format.h>
+#include <AK/Optional.h>
 #include <LibCore/ArgsParser.h>
 #include <LibCore/DateTime.h>
 #include <LibCore/File.h>
 #include <LibGUI/Application.h>
 #include <LibGUI/Clipboard.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/Widget.h>
+#include <LibGUI/Window.h>
 #include <LibGUI/WindowServerConnection.h>
 #include <LibGfx/PNGWriter.h>
 #include <unistd.h>
 
+class SelectableLayover final : public GUI::Widget {
+    C_OBJECT(SelectableLayover)
+public:
+    SelectableLayover(GUI::Window* window)
+        : m_window(window) {};
```

Comment:
Please move the empty block to a new line, and remove the extraneous `;`
```suggestion
    SelectableLayover(GUI::Window* window)
        : m_window(window)
    {
    }
```

Reply:
Done